src/main/java/com/epam/training/Main.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.training;
+
+import com.epam.training.model.ChristmasGifts;
+import com.epam.training.view.ChristmasGiftPrinter;
+
+public class Main {
+    public static void main(String[] args) {
+        ChristmasGiftCreator giftCreator = new ChristmasGiftCreator();
+        ChristmasGifts christmasGift = giftCreator.createGifts();
```

Comment:
better to have on create method that return all candies including subtypes. Or create to separate creators